src/views/html.ts

```diff
@@ -20,6 +20,13 @@ export function getHeader(options: {withCollapsed?: boolean} = {}): string {
       top: 0;           /* Don't forget this, required for the stickiness */
     }
 
+    tfoot tr {
+      background-color: rgb(128, 128, 128); /* same as var(--vscode-panelSectionHeader-background) but without transparancy */
```

Comment:
I don't think this is the right colour to use. I will share some screenshots, but it doesn't come through too well in Light Modern or Dark Modern like the header does. I also think we should use the CSS variables when we are able to instead of hardcoding colours.

https://code.visualstudio.com/api/references/theme-color

Reply:
Makes sense.  Did you have a specific CSS variable you'd suggest?

Reply:
`--vscode-multiDiffEditor-headerBackground` seems like a good option so far.

<img width="578" alt="image" src="https://github.com/codefori/vscode-db2i/assets/3708366/4ece6be4-7523-4804-bc33-59b0167d4e0f">

<img width="537" alt="image" src="https://github.com/codefori/vscode-db2i/assets/3708366/33cd1878-e153-492c-8f57-e1fee1e578d9">

<img width="614" alt="image" src="https://github.com/codefori/vscode-db2i/assets/3708366/560f17e9-478c-4714-b4c3-912e8a371b16">